config/locales/en.yml

```diff
@@ -889,10 +895,9 @@ en:
       heading: Your trainee teachers
       draft_heading: Draft trainees
       draft_trainees_link: View all draft trainees (%{count})
-      draft_apply_trainees_link: View draft trainees imported from Apply (%{count})
-      new_trainee_link: Create a new trainee record
+      draft_apply_trainees_link: View draft trainees imported from Apply (%{count} out of %{total} draft trainees)
```

Comment:
In the prototype, we only show this extra bit where the apply count is different than the draft count.

Reply:
Should be fixed now 👍 in this commit https://github.com/DFE-Digital/register-trainee-teachers/pull/2460/commits/893fbf2e27c1600a81f60fe45f89ec67aa4e3a23